runtime/gc_api/HeapIteratorAPI.cpp

```diff
@@ -466,13 +459,6 @@ j9mm_arraylet_identification(
 	*width = 0;
 	*mask = 0;
 	*result = 0;
-#else /* J9VM_GC_ARRAYLETS */
-	*arrayletLeafSize = 0;
-	*offset = 0;
-	*width = 0;
-	*mask = 0;
-	*result = 0;
-#endif /* J9VM_GC_ARRAYLETS */
```

Comment:
TODO: I can not find any caller of this function. If there is no any really it can be removed